packages/teuchos/comm/cmake/Dependencies.cmake

```diff
@@ -1,6 +1,7 @@
 TRIBITS_PACKAGE_DEFINE_DEPENDENCIES(
   LIB_REQUIRED_PACKAGES TeuchosCore TeuchosParameterList
-  )
+  LIB_OPTIONAL_TPLS magistrate
+)
 
```

Comment:
This is where you would need to put in a call to:

```cmake
tribits_tpl_tentatively_enable(magistrate)
```

See:

* https://tribitspub.github.io/TriBITS/users_guide/index.html#how-to-tentatively-enable-an-external-package-tpl